hooks/post-command

```diff
@@ -1,20 +1,19 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+# shellcheck source=lib/shared.bash
+source "$DIR/../lib/shared.bash"
+
 if [[ ${BUILDKITE_COMMAND_EXIT_STATUS:-0} != '0' ]]; then
   echo '~~~ Skipping S3 sync because the command failed'
   exit 0
 fi
 
-params=()
-
-if [[ "${BUILDKITE_PLUGIN_AWS_S3_SYNC_DELETE:-false}" == "true" ]]; then
-  params+=(--delete)
-fi
-
-if [[ "${BUILDKITE_PLUGIN_AWS_S3_SYNC_FOLLOW_SYMLINKS:-true}" == "false" ]]; then
-  params+=(--no-follow-symlinks)
+if [[ $BUILDKITE_PLUGIN_AWS_S3_SYNC_SOURCE != s3://* ]] ;then
+  aws_s3_sync "$BUILDKITE_PLUGIN_AWS_S3_SYNC_SOURCE" "$BUILDKITE_PLUGIN_AWS_S3_SYNC_DESTINATION"
+else
+  echo '~~~ Skipping S3 sync during post-command because source is s3'
```

Comment:
I'd prefer this didn't print as I'd like to cutdown on noise during the normal state of running (happy path). We can infer this code path given the absence of `~~~ :s3: Syncing $source to $destination` in the log.

```suggestion
```